examples/mixture/plot_gmm_selection.py

```diff
@@ -4,7 +4,8 @@
 ================================
 
 This example shows that model selection can be performed with
-Gaussian Mixture Models using information-theoretic criteria (BIC).
+Gaussian Mixture Models using
+:ref:`information-theoretic criteria <aic_bic>` (BIC).
```

Comment:
I think we do not need to wrap and still be under 88 characters:

```suggestion
Gaussian Mixture Models using :ref:`information-theoretic criteria (BIC) <aic_bic>`.
```

Also, I think we can include "BIC" in the link.